packages/core/scripts/mainnet/ProposeAdmin.js

```diff
@@ -31,9 +31,6 @@ async function propose(callback) {
      *********************************/
     const signingAccount = (await web3.eth.getAccounts())[0];
     console.group(`Proposer account: ${signingAccount}`);
-    if (signingAccount !== "0x2bAaA41d155ad8a4126184950B31F50A1513cE25") {
```

Comment:
This was unnecessary, as the address is checked against the role explicitly later in the script.

Reply:
@daywiss fixes this in another PR but that one might be blocked for a bit longer, +1